students/chieu_quach/lesson03/mailroom.py

```diff
@@ -3,6 +3,7 @@
 # Assignment -  Lesson 3
 # Exercise   -  Mailroom Part 1
 
+# mailroom part 1
```

Comment:
@Chieu1 - I'm unable to do inline comments on this file due to the issues with submitting this for review.  But, overall it works well.  There are a few PEP8 issues you'll want to address, like extra blank lines.  Take a look at installing a linter on you text editor to catch those.  Also, your import statements should all be at the top of the file.